stripe/src/main/java/com/stripe/android/view/CardNumberEditText.kt

```diff
@@ -174,7 +174,7 @@ class CardNumberEditText @JvmOverloads constructor(
                 if (formattedNumber != null) {
                     setText(formattedNumber)
                     newCursorPosition?.let {
-                        setSelection(it)
+                        setSelection(it.coerceIn(0, fieldText.length))
```

Comment:
Note that `coerceIn` returns values inclusive of the start and end values. Meaning this could return `fieldText.length`.

Haven't looked, but is the max value `setSelection` takes `n` or is it `n - 1`?

I also noticed a similar pattern in `ExpiryDateEditText`, does that need to be updated too?

Reply:
Thanks for pointing that out. This works, because the cursor position after the last character in the EditText will have index equal to the length of the contents.

Reply:
I'll apply the same logic to ExpiryDateEditText